pandas/io/parsers/base_parser.py

```diff
@@ -546,6 +548,65 @@ def _convert_to_ndarrays(
                 print(f"Filled {na_count} NA values in column {c!s}")
         return result
 
+    def _set_noconvert_dtype_columns(self, col_indices, names):
```

Comment:
can you type this in any way? (esp the return value) and add a doc-string

Reply:
Done